src/components/Card.js

```diff
@@ -145,14 +146,16 @@ export const Card = ({ direction = 'ltr' }) => {
               <input
                 type="checkbox"
                 id="hide-password-checkbox"
+                checked={checkbox}
```

Comment:
It looks like `checkbox` roughly tracks the `network.hidePassword` state. Is it possible to implement this without introducing the new `[checkbox, setCheckbox]` state?

e.g.
```js
checked={network.hidePassword}
```

Reply:
Yes , i will do that !